airflow/providers/__init__.py

```diff
@@ -14,3 +14,16 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+from airflow.config_templates.airflow_local_settings import DEFAULT_LOGGING_CONFIG, LOG_LEVEL
+from airflow.configuration import conf
+
+PROVIDERS_LOGGING_SETTINGS: str = conf.get('providers_google', 'VERBOSE_LOGGING')
```

Comment:
Should we set a fallback value?

Reply:
> Yes, can we move it to `airflow.providers.google.__init__.py` please?

Yes. it was mistake :(

Reply:
And yes fall back... I also have to make it differently to account for different logging  not only the default config (pointed out by @mik-laj